resources/com.sap.piper/pipeline/abapStageDefaults.yml

```diff
@@ -2,7 +2,8 @@ stages:
   Init: {}
   Initial Checks: {}
   Prepare System: {}
-  Clone Repositories: {}
+  Clone Repositories:
+    strategy: 'Pull'
```

Comment:
I noticed in another PR, that this default is not working (can be left out)

Reply:
I also wondered why there is a default in this file as well as in the groovy implementation where the strategy value gets added if the parameter is empty. Didn't check which one is getting handled but good to know! Removed it accordingly from the `resources/com.sap.piper/pipeline/abapStageDefaults.yml` file.